serverless/knative-serving/config-applications/multi-container-support-for-serving.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="multi-container-support-for-serving"]
+= Multi-container support for Serving
+:context: multi-container-support-for-serving
+
+You can deploy a multi-container pod using a single Knative service.
+This can be used, for example, to expose the service through a dedicated container, while keeping the main container more secure.
```

Comment:
To rephrase meant for breaking application responsibilities down to different parts.